src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -686,18 +686,31 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             const auto hr = DwmGetWindowAttribute(*_hostingHwnd, DWMWA_SYSTEMBACKDROP_TYPE, &attribute, sizeof(attribute));
             if (SUCCEEDED(hr))
             {
+                switch (attribute)
+                {
+                case DWMSBT_AUTO:
+                    OutputDebugStringW(L"AUTO\n");
+                    break;
+                case DWMSBT_MAINWINDOW:
+                    OutputDebugStringW(L"MAINWINDOW\n");
+                    break;
+                case DWMSBT_TABBEDWINDOW:
+                    OutputDebugStringW(L"TABBEDWINDOW\n");
+                    break;
+                case DWMSBT_TRANSIENTWINDOW:
+                    OutputDebugStringW(L"TRANSIENTWINDOW\n");
+                    break;
+                case DWMSBT_NONE:
+                    OutputDebugStringW(L"NONE\n");
+                    break;
+                }
                 isMicaAvailable = attribute == DWMSBT_MAINWINDOW;
             }
         }
-
-        if (!isMicaAvailable)
-        {
-            return;
-        }

```

Comment:
Okay, I think I see what's going on here. 

Instead of outright deleting this block, just move it below the `RequestedTheme(requestedTheme);` line. That way, we'll still set the `RequestedTheme` of the settings UI, regardless of the mica setting. 

Then, I'd make sure to test this with both mica enabled and disabled in the theme settings:

```jsonc
{
    "name": "mica",
    "window": 
    {
        "applicationTheme": "dark",
        "useMica": true
    },
    "settings": 
    {
        "theme": "light",
    },
    // etc...
},
```

(I haven't exactly tested this, but that's what my gut is saying)